fontc_crater/src/args.rs

```diff
@@ -1,9 +1,14 @@
 //! CLI args
 
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 
 use clap::{Parser, Subcommand};
 
+// this env var can be set by the runner in order to reuse git checkouts
+// between runs.
+static GIT_CACHE_DIR_VAR: &str = "CRATER_GIT_CACHE";
```

Comment:
It's not obvious why we need to probe an environment variable *and* support a cli option. Tbh, I'd prefer only cli arg but if we think we really need this lets document why.

Reply:
The reason this uses an env var is because in the CI environment we don't call this executable directly; we call the `run.sh` script in the `fontc_crater` repo, which in turn calls the `ci.sh` script in this repo, which executes this binary. The environment variable is set in the config for the scheduler that launches the CI task on the host machine.